src/witan/models/dem/ccm/fert/fertility_mvp.clj

```diff
@@ -115,8 +115,8 @@
    (cf/jumpoffyr-method-average historic-asfr
                                 :fert-rate
                                 :fert-rate
-                                1
-                                (inc fert-last-yr))})
+                                2013
+                                2014)})
```

Comment:
I think I'll change those into params like for mortality. Then I'll add schema validation for those as well!